iteration/iteration.go

```diff
@@ -253,6 +256,33 @@ func (m *GormIterationRepository) CanStart(ctx context.Context, i *Iteration) (b
 	return true, nil
 }
 
+func inTimeframe(startAt time.Time, endAt time.Time) bool {
+	return time.Now().UTC().After(startAt) && time.Now().UTC().Before(endAt)
+}
+
+func (i *Iteration) IsActive() bool {
+	if *i.UserActive == true {
```

Comment:
Simplify to `if *i.UserActive {` please. The compiler also tells you to do so.

Reply:
Will optimize IsActive(..)

Reply:
handled it here -> 1ecb8316bc9ca97b2c1adf5871d95c94326e553d